polyply/src/generate_templates.py

```diff
@@ -312,9 +312,9 @@ def _gen_templates(self, meta_molecule):
                     if success:
                         break
                     elif opt_counter > self.max_opt:
-                        LOGGER.warning(("Failed to optimize structure for block {}."
-                                        "Proceeding with unoptimized coordinates."
-                                        "Usually this is OK, but check your final structure.", resname))
+                        LOGGER.warning("Failed to optimize structure for block {}.", resname)
+                        LOGGER.warning("Proceeding with unoptimized coordinates.")
+                        LOGGER.warning("Usually this is OK, but check your final structure.""")
```

Comment:
Why? Not needed IMO

Reply:
It is good to give a warning about this, because the final structure might fail to energy minimize with infinite energy. In that case it could be either you are allowing to large overlap or something else. If this is printed it makes it easier to filter out.